README.markdown

```diff
@@ -1,14 +1,23 @@
 # How to compile (Linux)
 
-Make sure CMake and the development libraries for SDL, OpenAL and Vorbis are installed. In Debian-based systems, you need to install the following packages:
-
-    cmake libsdl1.2-dev libopenal-dev libvorbis-dev
-
-Then simply issue:
-
-    mkdir build && cd build
-    cmake ..
-    make
+Make sure the following dependencies are installed:
+
+* cmake
+* pkg-config
+* SDL2
+* openal
+* libvorbis
+* libpng
+
+Build using cmake:
+
+```sh
+mkdir build
+cd build
+cmake ..
+make
+./src/freegish
```

Comment:
This doesn't work, needs to be run from the root, not the build directory.

Reply:
Changed this around so that freegish indeed ends up in project root. Except when installing the project to FHS, then it won't clutter the project root.